README.md

```diff
@@ -120,7 +120,7 @@ kubectl create -f example/jsonnet-with-crd-as-config/grafana.yaml
 
 If the built in functionality is not sufficient, additional renderers, triggers and rollout actions can be injected with only few lines of code.
 
-TODO(brancz): explain how everything is a library, and main.go can be viewed as just a sample implementation.
+TODO(rollandf): explain how everything is a library, and main.go can be viewed as just a sample implementation.
```

Comment:
I'm happy if you want to do it, but not necessary :)

Reply:
Reverted